R/processes.R

```diff
@@ -344,3 +363,28 @@ create_lagged_eir <- function(variables, solvers, parameters) {
     }
   )
 }
+
+#' @title Hypnozoite decay function
+#' @description
+#' calulates the number of decaying hypnozoites in each individual
+#'
+#' @param variable the hypnozoite variable to update
+#' @param rate the hypnozoite decay rate
+#' @noRd
+create_hypnozoite_decay_process <- function(hypnozoites, gammal, renderer){
+  function(timestep){
+    if(hypnozoites$get_size_of(0)>0){
```

Comment:
Not sure I understand this line... does this not check if there are any individuals with 0 hypnozoites?

Reply:
Oh, good spot. This should be capturing if there are any individuals with hypnozoites. I believe it just needs an "!" to correct that. I'll fix it in the next commit.